### PR TITLE
(DOCSP-21612): [Swift] New page: How to use Realm in Xcode Playgrounds

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-21612

### Staged Changes (Requires MongoDB Corp SSO)

- [Quick Start - Realm in Xcode Playgrounds](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-21612/sdk/swift/xcode-playgrounds/): New page

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/swift/xcode-playgrounds.txt [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+   For more guidance on getting started with Realm and Swift or SwiftUI,
+   see any of these quick starts or tutorials:
+
+   - :ref:`Quick Start <ios-client-quick-start>`
+   - :ref:`Quick Start with {+sync-short+} <ios-client-quick-start-with-sync>`
+   - :ref:`Quick Start with SwiftUI <ios-swiftui-quick-start>`
+   - :ref:`iOS Tutorial <ios-swift-tutorial>`
+
+Within a project, go to :guilabel:`File` > :guilabel:`New` > 
+:guilabel:`Playground`. Select the type of Playground you want; for this example,
```
Comment (cbush) [inline source/sdk/swift/xcode-playgrounds.txt:36]:
```suggestion
:guilabel:`Playground`. Select the type of Playground you want. For this example,
```

File: source/sdk/swift/xcode-playgrounds.txt [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+   :lightbox:
+
+Name and save the playground in the root of your 
+project. Be sure to add it to the project:
+
+.. figure:: /images/xcode-playgrounds-create-in-project.png
+   :alt: Screenshot of the File Navigator with a Playground filename and the project selected
+   :lightbox:
+
+You should see your new Playground in your Project navigator.
```
Comment (cbush) [inline source/sdk/swift/xcode-playgrounds.txt:50]: Maybe? (And the one below)
```suggestion
You should see your new Playground in your Project Navigator.
```

**Review by cbush — APPROVED** [review]
Looks good!

File: source/sdk/swift/xcode-playgrounds.txt (hunk 2 above)
Comment (dacharyc) [reply source/sdk/swift/xcode-playgrounds.txt:50]: I went with Apple's documentation for guidance on this, and it seems `Project navigator` is the capitalization they use in their docs: https://developer.apple.com/documentation/xcode/managing-files-and-folders-in-your-xcode-project

**Comment by dacharyc** [comment]
Failing test is in FlexibleSync tests, not related to this PR. I'll create a ticket to update it next week, meanwhile merging this PR.

**Pull request merged**